guides/tutorials/agent-config.md

```diff
@@ -114,7 +114,7 @@ accepted.
 **Type**: `string`
 
 ```typescript title="example"
-endpoint: ["https://example.org:3000"]
+endpoint: ["https://example.org:3000"];
```

Comment:
these changes seem unrelated.

@blu3beri I also ran into problems with formatting. Can you look at adding a prettier config so we avoid these formatting changes?

Reply:
ah yes!